packages/utils/telemetry-utils/src/test/config.spec.ts

```diff
@@ -58,18 +58,17 @@ describe("Config", () => {
 		const config = new CachedConfigProvider(logger, inMemoryConfigProvider(mockStore));
 
 		assert.equal(config.getNumber("number"), 1);
-		assert(
-			logger.matchEvents([
-				{
-					eventName: "ConfigRead",
-					configName: { tag: TelemetryDataTag.CodeArtifact, value: "number" },
-					configValue: {
-						tag: TelemetryDataTag.CodeArtifact,
-						value: `{"raw":"1","string":"1","number":1}`,
-					},
+		logger.assertMatch([
+			{
+				category: "generic",
+				eventName: "ConfigRead",
+				configName: { tag: TelemetryDataTag.CodeArtifact, value: "number" },
```

Comment:
Why not use `tagCodeArtifacts`?

Reply:
see above. we should explicitly match in tests